Packet++/header/TcpReassembly.h

```diff
@@ -166,18 +166,25 @@ class TcpStreamData
 	 */
 	const ConnectionData& getConnectionData() const { return m_Connection; }
 
-	/**
+	/** 
+	 * DEPRECATED
```

Comment:
We can use the deprecation warning we do here: https://github.com/seladb/PcapPlusPlus/blob/22c6c85130ececa0cd4f9367332b8142687722c4/Common%2B%2B/header/IpAddress.h#L151